index.liquid

```diff
@@ -52,18 +52,26 @@
                         <dd><p>{{group.description}}</p><dd>
                     {% endfor %}
                 </dl>
+                <dl class="general-public">
+                <dt><h3>General Public Opening Times</h3><dt>
+                    {% for opening in supermarket.generalHours %}
+                        <dt><strong><p>{{opening.day}}</p></strong><dt>
```

Comment:
Note sure what happens here design wise so may need to tweak but we shouldn't need `<strong>` as semantic meaning comes from the `dt` and the paragraph I think may be not required. Note: for *styling* only (where you don't need the semantic meaning) `<b>` is better.
```suggestion
                        <dt>{{opening.day}}<dt>
```